### PR TITLE
Add Onboarding Files

### PR DESCRIPTION
**Pull request by Avikaa09**
Onboarding Portal

### CODE REVIEW

File: src/pages/Restaurants/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { Card, Space, Statistic } from "antd";
+import { useState } from "react";
+import './index.css';
+import { Link } from "react-router-dom";
+
+function Restaurants(){
```
Comment (amosaxe) [inline src/pages/Restaurants/index.js:6]: Have we changed this to handle dynamic data?

File: src/pages/Offers/index.js [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+                <Option value="percentage">Percentage</Option>
+                <Option value="fixed">Fixed</Option>
+              </Select>
+            </Form.Item>
+            <Form.Item
+              label="Max Value"
+              name="maxValue"
+              rules={[{ required: true, message: 'Please enter the maximum value' }]}
+            >
+              <InputNumber />
```
Comment (amosaxe) [inline src/pages/Offers/index.js:105]: Please add validation checks.

File: src/pages/Offers/index.js [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+                { required: true, message: 'Please enter the usage limits' },
+              ]}
+            >
+              <Input.Group compact>
+                <Form.Item
+                  name={['usageLimits', 'maxUses']}
+                  noStyle
+                  rules={[{ required: true, message: 'Please enter the maximum uses' }]}
+                >
+                  <InputNumber placeholder="Max Uses" style={{ width: '50%' }} />
```
Comment (amosaxe) [inline src/pages/Offers/index.js:139]: Have we checked responsiveness?

File: src/pages/Restaurants/index.js (hunk 1 above)
Comment (amosaxe) [inline src/pages/Restaurants/index.js:6]: Have we changed code to handle dynamic data?

File: src/pages/Restaurants/index.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    const handleTitleClick = () => {
+        setExpanded(!expanded);
+      };
+    return(
+        
+        <Card onClick={handleTitleClick} className={`restaurant-card ${expanded ? "expanded" : ""}`}>
+        <Card.Meta title={title} />
+        {expanded && (
+          <Space direction="vertical">
+            {/* <Statistic title="Value" value={value} /> */}
```
Comment (amosaxe) [inline src/pages/Restaurants/index.js:52]: remove comments

File: src/pages/Subscription/Subscription.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+        </>
+            )
+        }
+
+
+var columns = [
+  {
+    title: 'CustomerID',
+    dataIndex: 'CustomerID',
+
```
Comment (amosaxe) [inline src/pages/Subscription/Subscription.js:18]: Dont give extra lines

File: src/pages/Offers/index.js [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+            </Form.Item>
+            <Form.Item
+              label="Min Order Amount"
+              name="minOrderAmount"
+              rules={[
+                { required: false, message: 'Please enter the minimum order amount' },
+              ]}
+            >
+              <InputNumber />
+            </Form.Item>
```
Comment (amosaxe) [inline src/pages/Offers/index.js:239]: I dont think we have added confirmations before submission?

File: src/pages/Analytics/CAnalytics.js [hunk 7]
```diff
@@ -0,0 +1,94 @@
+import React from 'react';
```
Comment (amosaxe) [inline src/pages/Analytics/CAnalytics.js:1]: change logo.svg

File: src/components/AppRoutes/index.js [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+       <Route path="/Restaurants" element={<Restaurants/>}/>
+       <Route path="/Subscription" element={<Subscription/>}/>
+       <Route path="/AddRestaurant" element={<AddRestaurant/>}/>
+    </Routes>
+
+  
+)
+}
+
+export default AppRoutes;
```
Comment (amosaxe) [inline src/components/AppRoutes/index.js:29]: add empty line at the end of all files.

File: src/components/Content/index.js [hunk 9]
```diff
@@ -0,0 +1,9 @@
+import AppRoutes from "../AppRoutes";
+
+function Content(){
+    return <div className="content">
```
Comment (amosaxe) [inline src/components/Content/index.js:4]: Follow formatting

File: src/components/AddRestaurant.js [hunk 10]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function AddRestaurant(){
+    return(
+        <div>
+            <h1>Add Restaurant</h1>
+
```
Comment (amosaxe) [inline src/components/AddRestaurant.js:7]: no extra lines

File: src/pages/Offers/index.js [hunk 11]
```diff
@@ -0,0 +1,314 @@
...
+                )}
+              </li>
+            ))}
+          </ul>
+        </div>
+      )}
+    </div>
+  );
+};
+
```
Comment (amosaxe) [inline src/pages/Offers/index.js:313]: Add confirmations before submission(basically creating objects)

File: src/pages/Analytics/RAnalytics.js [hunk 12]
```diff
@@ -0,0 +1,229 @@
...
+    title: {
+      display: true,
+    //   text: 'Orders Monthly',
+    },
+  },
+};
+const monthlyLabels = ['January', 'February', 'March', 'April', 'May', 'June', 'July'];
+<canvas role="img" height="400" width="375" style="box-sizing: border-box; display: block; height: 150px; width: 300px;"></canvas>
+export const monthlyData = {
+  labels: monthlyLabels,
```
Comment (amosaxe) [inline src/pages/Analytics/RAnalytics.js:41]: please have dummy data as same as expected from the backend.
Or
add code to convert the data from backend into this format

File: src/pages/Analytics/CAnalytics.js [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+      }
+    });
+  
+    return (
+      <>
+        <Table columns={columns} dataSource={customerData} />
+        {maxOrdersCustomer && (
+          <div>
+           <h3> Customer with Maximum Orders: </h3>{maxOrdersCustomer.CustomerID} (Orders: {maxOrdersCustomer.NumberofOrders})
+          </div>
```
Comment (amosaxe) [inline src/pages/Analytics/CAnalytics.js:86]: We need customer name and other info.
Please align with ashutosh on this once. I think we want top 5 customers?

File: src/components/Sidebar/index.css [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+   
+    flex-direction: column;
+    justify-content:flex-start; 
+    align-items: stretch; 
+    width: 100%S;
+  }
+ 
+  
+  .ant-menu-item,
+.ant-menu-submenu-title {
```
Comment (amosaxe) [inline src/components/Sidebar/index.css:24]: correct the formatting

**Review by amosaxe — CHANGES_REQUESTED** [review]
Please go through all the comments. Make sure that they are followed throughout the project.

File: src/components/Sidebar/index.css (hunk 14 above)
Comment (Avikaa09) [reply src/components/Sidebar/index.css:24]: done

File: src/components/AddRestaurant.js (hunk 10 above)
Comment (Avikaa09) [reply src/components/AddRestaurant.js:7]: done

File: src/pages/Subscription/Subscription.js (hunk 5 above)
Comment (Avikaa09) [reply src/pages/Subscription/Subscription.js:18]: done

File: src/pages/Restaurants/index.js (hunk 4 above)
Comment (Avikaa09) [reply src/pages/Restaurants/index.js:52]: done

File: src/pages/Offers/index.js (hunk 2 above)
Comment (Avikaa09) [reply src/pages/Offers/index.js:105]: done

File: src/pages/Analytics/RAnalytics.js (hunk 12 above)
Comment (Avikaa09) [reply src/pages/Analytics/RAnalytics.js:41]: We'll see

**Pull request merged**